### PR TITLE
Update Team Information

### PR DESCRIPTION
**Pull request by lvora105**
- Included team images 
- took out some Google formatting
- Inserted team description from slides

### CODE REVIEW

**Comment by aaronlippold** [comment]
remove placeholder text

**Comment by aaronlippold** [comment]
Move Brett to the front of the list

**Comment by aaronlippold** [comment]
Use MITRE titles for sub-titles on cards - add MITRE org

File: src/assets/data/Team.json [hunk 1]
```diff
@@ -1,62 +1,98 @@
...
+    "image": "aaron-lippold.png"
+  },
+  {
+    "name": "Eugene Aronne",
+    "designation": "Project Lead",
+    "bio": "Lorem ipsum dolor, sit amet consectetur adipisicing elit. Adipisci est, eius non minima amet quaerat expedita quos exercitationem quia aspernatur blanditiis totam officia commodi eos sapiente natus excepturi ab nemo.",
+    "image": "eugene-aronne.png"
+  },
+  {
+    "name": "Brett Kreider",
```
Comment (aaronlippold) [inline src/assets/data/Team.json]: Brett should be first in the list

File: src/assets/data/Team.json [hunk 2]
```diff
@@ -1,62 +1,98 @@
 [
   {
-    "name": "Name",
+    "name": "Aaron Lippold",
+    "designation": "Project Lead",
```
Comment (aaronlippold) [inline src/assets/data/Team.json]: This should be the MII Role 'Cyber Security Assessor' or 'Principal Cyber Security Engineer'

File: src/assets/data/Team.json [hunk 3]
```diff
@@ -1,62 +1,98 @@
...
+    "bio": "Lorem ipsum dolor, sit amet consectetur adipisicing elit. Adipisci est, eius non minima amet quaerat expedita quos exercitationem quia aspernatur blanditiis totam officia commodi eos sapiente natus excepturi ab nemo.",
+    "image": "aaron-lippold.png"
```
Comment (aaronlippold) [inline src/assets/data/Team.json]: add `roles`: ['Profile Development', 'InSpec_Tools Dev', 'Heimdall-Lite Lead']

File: src/assets/data/Team.json (hunk 1 above)
Comment (lvora105) [reply src/assets/data/Team.json]: done!

File: src/assets/data/Team.json (hunk 2 above)
Comment (lvora105) [reply src/assets/data/Team.json]: Done and edited per stand up notes

File: src/assets/data/Team.json (hunk 3 above)
Comment (lvora105) [reply src/assets/data/Team.json]: done!

**Pull request merged**